portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -410,6 +412,67 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 	}
 </script>
 
+<script type="text/javascript">
```

Comment:
You can add the `data-senna-track="temporary"` attribute to this tag so that Senna.js does not move this script to other pages when navigating through DXP.

Reply:
Fixed